test/openAPI/features/support/db_read.js

```diff
@@ -1,177 +1,77 @@
 When(
-  'The user sends an invalid request without routing parameters to view a database schema version',
-  () =>
+  'User sends GET request with given Information-Mediator-Client header and {string} as id',
+  id =>
     specDatabaseRead
       .get(baseUrl)
       .withHeaders(`${header.key}`, `${header.value}`)
+      .withPathParams({ id: id })
 );
 
 Then(
-  'The result of an operation to receive database information from Digital Registries with schema version returns an invalid route param error',
-  async () => {
-    await specDatabaseRead.toss();
-    specDatabaseRead.response().should.have.status(400);
-    specDatabaseRead.response().should.have.body('{"Invalid route param id"}');
-  }
-);
-
-// Scenario: The user is unable to receive database information from Digital Registries with schema version because no header is included
-When(
-  'The user sends an invalid request to view a database schema version',
-  () => specDatabaseRead.get(baseUrl).withHeaders(`${header.key}`, '')
+  'User receives a response from the \\/database\\/id endpoint',
+  async () => await specDatabaseRead.toss()
 );
 
 Then(
-  'The result of an operation to receive database information from Digital Registries with the schema version returns an error',
-  async () => {
-    await specDatabaseRead.toss();
-    specDatabaseRead.response().should.have.status(400);
+  'The \\/database\\/id endpoint response should be returned in a timely manner 15000ms',
+  () =>
     specDatabaseRead
       .response()
-      .should.have.body(
-        '{"Invalid format of Information-Mediator-Client, should match INSTANCE/CLASS/MEMBER/SUBSYSTEM"}'
-      );
-  }
+      .to.have.responseTimeLessThan(defaultExpectedResponseTime)
 );
 
-// Scenario: The user cannot receive Digital Registries database information with the schema version because there is no header included
-When(
-  'The user sends an invalid request with no payload to view a database schema version',
-  () => specDatabaseRead.get(baseUrl)
+Then('The \\/database\\/id endpoint response should have status 200', () =>
+  specDatabaseRead.response().to.have.status(200)
 );
 
 Then(
-  'The result of receiving Digital Registries database information with the schema version is an error',
-  async () => {
-    await specDatabaseRead.toss();
-    specDatabaseRead.response().should.have.status(400);
+  'The \\/database\\/id endpoint response should have content-type: application\\/json header',
+  () =>
     specDatabaseRead
       .response()
-      .should.have.body('{"Information-Mediator-Client required"}');
-  }
+      .should.have.header(acceptHeader.key, acceptHeader.value)
 );
 
-After(() => {
+Then('The \\/database\\/id endpoint response should match json schema', () =>
+  chai
+    .expect(specDatabaseRead._response.json)
+    .to.be.jsonSchema(databaseReadResponseSchema)
+);
+
+//Scenario Outline: User successfully obtains Digital Registries database information with schema version smoke type test
```

Comment:
without smoke type test - and if you apply suggestion from feature file it will be to change accordingly